src/Robot.MessageBus/Messages/AbstractVectorMessage.cs

```diff
@@ -2,8 +2,7 @@
 
 namespace Robot.MessageBus.Messages
 {
-    [Topic(Topics.Motion.Velocity)]
```

Comment:
Every Message should have a Topic Attribute as that is what routes messages

Reply:
Thanks for noticing that. This one's abstract and has two implementations: RequestedVelocityMessage and AppliedVelocityMessage, both of them have the Topic attribute.

Reply:
Ahh  ok ok I didnt notice that